### PR TITLE
openApiUtil test

### PR DESCRIPTION
**Pull request by jinyangyang222**

### CODE REVIEW

File: modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import org.eclipse.xpanse.modules.models.utils.OclLoader;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test for OpenApiUtilTest.
+ */
+public class OpenApiUtilTest {
+
+    @Disabled
```
Comment (baixinsui) [inline modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java:26]: Please ensure that the Test class is correct and valid.

File: modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        String s = openApiUtil.creatServiceApi(registerServiceEntity);
+        System.out.println(s);
+    }
+
+    @Test
+    public void deleteServiceApi_tese() throws Exception {
+        String id = "624e2a47-b1be-414d-b8c5-967e09c13dfe";
+        DeployVariableValidator deployVariableValidator = new DeployVariableValidator();
+        OpenApiUtil openApiUtil = new OpenApiUtil(deployVariableValidator);
+        openApiUtil.deleteServiceApi(id);
```
Comment (baixinsui) [inline modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java:67]: same as above.

File: modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java (hunk 1 above)
Comment (jinyangyang222) [reply modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java:26]: ok, i do it

File: modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java (hunk 2 above)
Comment (jinyangyang222) [reply modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java:67]: I have adjusted

File: modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        registerServiceEntity.setId(UUID.randomUUID());
+        registerServiceEntity.setName("kafka");
+        registerServiceEntity.setVersion("1.0");
+        registerServiceEntity.setCsp(Csp.HUAWEI);
+        registerServiceEntity.setCategory(Category.AI);
+        registerServiceEntity.setOcl(ocl);
+        registerServiceEntity.setServiceState(ServiceState.REGISTERED);
+        DeployVariableValidator deployVariableValidator = new DeployVariableValidator();
+        OpenApiUtil openApiUtil = new OpenApiUtil(deployVariableValidator);
+        openApiUtil.updateServiceApi(registerServiceEntity);
```
[the remark this answers is not in the document]
Comment (jinyangyang222) [reply modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java:41]: ok,I have adjusted

File: modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        registerServiceEntity.setName("kafka2");
+        registerServiceEntity.setVersion("2.0");
+        registerServiceEntity.setCsp(Csp.HUAWEI);
+        registerServiceEntity.setCategory(Category.AI);
+        registerServiceEntity.setOcl(ocl);
+        registerServiceEntity.setServiceState(ServiceState.REGISTERED);
+        DeployVariableValidator deployVariableValidator = new DeployVariableValidator();
+        OpenApiUtil openApiUtil = new OpenApiUtil(deployVariableValidator);
+        String s = openApiUtil.creatServiceApi(registerServiceEntity);
+        System.out.println(s);
```
[the remark this answers is not in the document]
Comment (jinyangyang222) [reply modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtilTest.java:59]: ok, i will use log

**Comment by baixinsui** [comment]
Looks good.

**Pull request merged**